compliance/tests/goroutines/expected.log

```diff
@@ -1,11 +1,10 @@
 Main: Starting workers
 Main: Workers started
-Anonymous function worker
 Worker 0 starting
 Worker 0 done
 Worker 1 starting
 Worker 1 done
-Another worker: test
 Worker 2 starting
 Worker 2 done
-Main: All workers completed
\ No newline at end of file
+Another worker: test
+Anonymous function worker
```

Comment:
Delete this file, modify the .go file to have deterministic log order without the race conditions, and rerun the tests, then git add all and commit and push.